os/include/tinyara/mm/mm.h

```diff
@@ -302,21 +302,21 @@ struct mm_allocnode_s {
 /* This describes a free chunk */
 
 struct mm_freenode_s {
-	mmsize_t size;				/* Size of this chunk */
 	mmsize_t preceding;			/* Size of the preceding chunk */
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
```

Comment:
Please check that other tools which related to heap like heapinfo works fine or not.